package.json

```diff
@@ -4,25 +4,24 @@
   "author": "Workday, Inc. (https://www.workday.com)",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/core": "^7.2.2",
+    "@babel/core": "^7.8.0",
     "@babel/plugin-proposal-class-properties": "^7.3.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.4.4",
-    "@babel/preset-env": "^7.3.1",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-env": "^7.8.0",
+    "@babel/preset-react": "^7.8.0",
     "@babel/preset-typescript": "^7.1.0",
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.1.0",
     "@emotion/is-prop-valid": "^0.8.2",
-    "@storybook/addon-actions": "^5.3.14",
-    "@storybook/addon-docs": "^5.3.17",
-    "@storybook/addon-knobs": "^5.3.14",
-    "@storybook/addon-storysource": "^5.3.14",
-    "@storybook/addon-viewport": "^5.3.14",
-    "@storybook/components": "^5.3.14",
-    "@storybook/react": "^5.3.17",
-    "@storybook/source-loader": "^5.3.14",
+    "@storybook/addon-actions": "^5.3.18",
+    "@storybook/addon-docs": "^5.3.18",
+    "@storybook/addon-knobs": "^5.3.18",
+    "@storybook/addon-storysource": "^5.3.18",
+    "@storybook/addon-viewport": "^5.3.18",
+    "@storybook/components": "^5.3.18",
+    "@storybook/react": "^5.3.18",
+    "@storybook/source-loader": "^5.3.18",
```

Comment:
Some of these monorepo dependencies were on `5.3.14` and others on `5.3.17`. I just updated to all be latest